x/evm/keeper/state.go

```diff
@@ -3,25 +3,37 @@ package keeper
 import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/state"
 	"github.com/sei-protocol/sei-chain/x/evm/types"
 )
 
 func (k *Keeper) GetState(ctx sdk.Context, addr common.Address, hash common.Hash) common.Hash {
 	val := k.PrefixStore(ctx, types.StateKey(addr)).Get(hash[:])
 	if val == nil {
-		if k.EthReplayConfig.Enabled {
-			// try to get from eth DB
-			tr, err := k.DB.OpenStorageTrie(k.Root, addr, common.BytesToHash(k.PrefixStore(ctx, types.ReplaySeenAddrPrefix).Get(addr[:])), k.Trie)
+		if !k.EthReplayConfig.Enabled && !k.EthBlockTestConfig.Enabled {
```

Comment:
we don't need to change this part right since we are genesis-ing into seidb directly